tests/data/sasaki_metric_data.py

```diff
@@ -26,7 +26,9 @@ class SasakiMetricTestData(TestData):
     def inner_product_test_data(self):
         _sqrt2 = 1.0 / gs.sqrt(2.0)
         base_point = gs.array([[_sqrt2, -_sqrt2, 0], [_sqrt2, _sqrt2, 1]])
-        _log = self.sas_sphere_metric.log(gs.array([self.pu0, self.pu1]), base_point)
+        end_point = gs.stack([self.pu0, self.pu1])
```

Comment:
NIT: what is pu0, pu1? what is sqrt2? the names could be more self explanatory.

Reply:
This comes from a very old PR. I'll take that into account in the new tests.